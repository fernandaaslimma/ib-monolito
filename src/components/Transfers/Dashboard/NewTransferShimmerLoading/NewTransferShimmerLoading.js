import React from "react";
import ShimmerLoading from "../../../common/ShimmerLoading";

import { Content, Fieldset, ButtonWrapper, Button } from "../styles";
import Card from "../../../common/Card";
import { translate } from "../../../../utils/i18n";
import { darkGreen } from "../../../../styles/settings";

function NewTransferShimmerLoading() {
  return (
    <Content>
      <Card BigTitle={translate("NEW_TRANSFER")} titleColor={darkGreen}>
        <Fieldset width={22} loading>
          <ShimmerLoading />
          <ShimmerLoading />
          <ShimmerLoading />
          <ShimmerLoading />
        </Fieldset>
        <Fieldset width={23} loading>
          <ShimmerLoading />
          <ShimmerLoading />
          <ShimmerLoading />
          <ShimmerLoading />
        </Fieldset>
        <Fieldset width={55} loading>
          <ShimmerLoading />
          <ShimmerLoading />
          <ShimmerLoading />
          <ShimmerLoading />
        </Fieldset>
        <ButtonWrapper>
          <Button disabled>
            <ShimmerLoading />
          </Button>
        </ButtonWrapper>
      </Card>
    </Content>
  );
}

export default NewTransferShimmerLoading;
