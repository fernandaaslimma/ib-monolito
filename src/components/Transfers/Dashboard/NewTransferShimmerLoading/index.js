import NewTransferShimmerLoading from "./NewTransferShimmerLoading";

export default NewTransferShimmerLoading;
