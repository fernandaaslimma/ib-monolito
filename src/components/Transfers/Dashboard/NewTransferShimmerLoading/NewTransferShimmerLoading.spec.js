import React from "react";
import { shallow } from "enzyme";
import NewTransferShimmerLoading from "./NewTransferShimmerLoading";

jest.mock("../../../../utils/i18n", () => ({
  translate: id => id,
  getLanguage: () => "en-US",
  isZhCN: () => false,
  isPtBR: () => true
}));

describe("NewTransferShimmerLoading", () => {
  it("should match snapshot", () => {
    expect(shallow(<NewTransferShimmerLoading />)).toMatchSnapshot();
  });
});
