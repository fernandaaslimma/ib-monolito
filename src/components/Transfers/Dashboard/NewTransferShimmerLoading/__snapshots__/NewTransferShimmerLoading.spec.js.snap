// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewTransferShimmerLoading should match snapshot 1`] = `
<styled.div>
  <Card
    BigTitle="NEW_TRANSFER"
    icon={null}
    iconColor={null}
    title={null}
    titleColor="#27445f"
  >
    <styled.div
      loading={true}
      width={22}
    >
      <ShimmerLoading
        darker={false}
        index={0}
        inverse={false}
      />
      <ShimmerLoading
        darker={false}
        index={0}
        inverse={false}
      />
      <ShimmerLoading
        darker={false}
        index={0}
        inverse={false}
      />
      <ShimmerLoading
        darker={false}
        index={0}
        inverse={false}
      />
    </styled.div>
    <styled.div
      loading={true}
      width={23}
    >
      <ShimmerLoading
        darker={false}
        index={0}
        inverse={false}
      />
      <ShimmerLoading
        darker={false}
        index={0}
        inverse={false}
      />
      <ShimmerLoading
        darker={false}
        index={0}
        inverse={false}
      />
      <ShimmerLoading
        darker={false}
        index={0}
        inverse={false}
      />
    </styled.div>
    <styled.div
      loading={true}
      width={55}
    >
      <ShimmerLoading
        darker={false}
        index={0}
        inverse={false}
      />
      <ShimmerLoading
        darker={false}
        index={0}
        inverse={false}
      />
      <ShimmerLoading
        darker={false}
        index={0}
        inverse={false}
      />
      <ShimmerLoading
        darker={false}
        index={0}
        inverse={false}
      />
    </styled.div>
    <styled.div>
      <Styled(Button)
        disabled={true}
      >
        <ShimmerLoading
          darker={false}
          index={0}
          inverse={false}
        />
      </Styled(Button)>
    </styled.div>
  </Card>
</styled.div>
`;
